From 5d29f7dae3bdeaa13253360125ebf11c5575bcd8 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 14 Dec 2019 23:46:34 +0200
Subject: [PATCH 3/4] remove 'su' from none-su builds (exactly, all)

  * quickfix, need another way to implement checking

	modified:   target/product/base_system.mk
---
 target/product/base_system.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index e3e3fbf7e..41a367898 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -361,7 +361,6 @@ PRODUCT_PACKAGES_DEBUG := \
     sqlite3 \
     ss \
     strace \
-    su \
     sanitizer-status \
     tracepath \
     tracepath6 \
-- 
2.11.0

